database/seeds/FreshInstallSeeder.php

```diff
@@ -15,9 +15,9 @@ public function run()
         ]);
 
         $tracks = [
-            'New to Programming',
-            'Frontend Developers',
-            'WordPress Developers',
+            'Iniciante na Programação',
+            'Programador Frontend',
+            'Programador WordPress',
```

Comment:
In seeds should not change, i.e. translate

Reply:
Agreed with @olsza

Reply:
ok @joseborges, I already know why you translated it ;)
I analyzed the code, and if you want to translate this content, you need to add it to your translation file in JSON. In the JSON file in your language you have to put something like this:
```
{
...
"New to Programming": "Iniciante na Programação",
"Frontend Developers": "Programador Frontend",
"WordPress Developers": "Programador WordPress",
...
}
```

thanks to you, I have to put it in the PL language file :)